src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -336,6 +341,10 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('mime_type.xml');
         }
 
+        if (class_exists(Mailer::class)) {
+            $loader->load('mailer.xml');
```

Comment:
can be removed as it's already loaded in `registerMailerConfiguration`

Reply:
indeed, done